src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -706,10 +746,10 @@ bamm:EntityInstancesHaveOnlyKnownProperties
    ] ;
    sh:sparql [
       a sh:SPARQLConstraint ;
-      sh:message "The Entity instance has an attribute that is not a Property defined in the Entity" ;
+      sh:message "Entity instance '{$this}' has an attribute '{?value}' that is not a Property defined in the Entity" ;
```

Comment:
"Entity instance '{$this}' has an attribute '{?value}' that is not a Property defined in the Entity."